ccd-definition/ComplexTypes.json

```diff
@@ -1556,5 +1556,257 @@
     "FieldType": "Document",
     "ElementLabel": "Upload a file",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "partyChild",
+    "ListElementCode": "partyID",
+    "FieldType": "Text",
+    "ElementLabel": "Party ID",
+    "FieldShowCondition": "partyType=\"DO_NOT_SHOW\"",
```

Comment:
this being hidden means data never gets persisted to the db. Working on the AboutToSubmitCallback still. Just struggling with mapping exceptions

Reply:
Yep, I fully intent to 'borrow' your solution ;)